BotWraper.ts

```diff
@@ -0,0 +1,129 @@
+import { ChildProcess, spawn } from 'node:child_process';
+import { stringify } from 'node:querystring';
+import { Queue } from 'queue-typescript';
+
+export enum ErrorCode {
+    Success,
+    NonZeroExitCode,
+    TLE,
+    UnexpectedExitOfCode
+}
+
+export class Data {
+    id: number;
+    data?: string;
```

Comment:
```suggestion
    data: string | null;
```
Az, hogy TLE-nél `data: null`-t adsz vissza, az tök jó. A null egy explicit semmi, az undefined meg csak úgy "épp nem jutott oda adat".
Viszont itt a ? lényegében `string | undefined`-ot jelent, strict módban szól is miatta a compiler.
https://www.typescriptlang.org/docs/handbook/2/objects.html#optional-properties